docs/sql-migration-guide.md

```diff
@@ -83,21 +83,22 @@ license: |
     * `\u000B` (vertical tab)
     * `\u0007` (bell)
 
-  - In Spark 3.2, `ALTER TABLE .. RENAME TO PARTITION` throws `PartitionAlreadyExistsException` instead of `AnalysisException` for tables from Hive external when the target partition already exists.
 
```

Comment:
Add an new blank line here since the following two content have no blank lines on the page
<img width="973" alt="截屏2022-03-03 上午11 16 20" src="https://user-images.githubusercontent.com/46485123/156489446-b85b248e-e533-4fb9-a925-6b9985469c49.png">